### PR TITLE
czg 1.3.8 (new formula)

### PR DESCRIPTION
**Pull request by Zhengqbbb**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- [x] `brew install --build-from-source czg`
- [x] `brew test czg`
- [x] `brew style czg`
- [x] `brew audit --new czg`
- [x] `brew audit --strict czg`

### CODE REVIEW

File: Formula/czg.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
+class Czg < Formula
+  desc "Interactive Commitizen CLI that generate standardized commit messages"
+  homepage "https://cz-git.qbenben.com/cli/"
+  url "https://registry.npmjs.org/czg/-/czg-1.3.8.tgz"
+  sha256 "c1994df8ff1829953b31e071101a8eea8fa5065fb95842afe33440a415bf6b93"
+  license "MIT"
+  head "https://github.com/Zhengqbbb/cz-git.git", branch: "main"
```
Comment (chenrui333) [inline Formula/czg.rb:9]: The head build does not work
```suggestion
```

Comment (Zhengqbbb) [reply Formula/czg.rb:9]: ok should I remove this line "head"?
I'm not sure if homebrew will pull the latest version from npm, "https://registry.npmjs.org"

Comment (chenrui333) [reply Formula/czg.rb]: yeah, you should remove that line

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@Zhengqbbb, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by Zhengqbbb** [comment]
> @Zhengqbbb, thanks for your contribution to Homebrew! 🎉 🥇
> 
> Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

@chenrui333 
Thanks for your review, **fast and responsible**. I will go on try my best to do a the best git commit CLI

**Pull request closed without merging**